### PR TITLE
Added auth token management to view and remove github auth tokens

### PR DESCRIPTION
**Pull request by BRIMIL01**

### CODE REVIEW

File: features/manage_auth_tokens.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Feature: Auth Token Management
+
+  In order to maange all of my auth token grants
```
Comment (cyphactor) [inline features/manage_auth_tokens.feature]: "maange" should be "manage"

File: features/manage_auth_tokens.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Feature: Auth Token Management
+
```
Comment (cyphactor) [inline features/manage_auth_tokens.feature]: Don't normally have a blank line here.

**Comment by cyphactor** [comment]
General question is what is the intended visibility of this feature? The reason I ask is because it is not currently navigable via the site. However, it is accessible to get to what you tested in cucumber via url hacking. If you just wanted to get this chunk in before exposing this to a user.

It is approved pending the small textual changes.

If that was not the intent add navigation to make it visible to normal users.

**Comment by BRIMIL01** [comment]
The intent was to add functionality in small iterations and progress to a point where account management can be exposed.

**Comment by cyphactor** [comment]
:+1:

**Pull request merged**